SendEmail/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Communication.Email;
+using Azure.Communication.Email.Models;
+
+namespace SendEmail
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            // This code demonstrates how to send email using Azure Communication Services.
+            string connectionString = "<ACS_CONNECTION_STRING>";
+            EmailClient emailClient = new EmailClient(connectionString);
+            string subject = "Send email quick start - dotnet";
+            var emailContent = new EmailContent(subject)
+            {
+                PlainText = "This is plain mail send test body \n Best Wishes!!",
+                Html = "<html><body><h1>Quick send email test</h1><br/><h4>Communication email as a service mail send app working properly</h4><p>Happy Learning!!</p></body></html>"
```

Comment:
>Html [](http://example.com/codeflow?start=16&length=4)

I'd suggest having a clickable link also in the html body, so one can test the link click User Engagement tracking.

Reply:
Can we not track uses by git downloads, setting a link into email might be some privacy/security concern. Thoughts ..